src/forms/CalendarComponent.jsx

```diff
@@ -49,13 +49,10 @@ export class CalendarComponent extends React.Component {
 		} = this.props;
 
 		const classNames = {
-			label: cx(
-				{
-					'label--required': required,
-					'flush--bottom': helperText,
-				},
-				labelClassName
-			),
+			label: cx({
+				required,
```

Comment:
The class name should be `label--required` here. There may be a `required` class in app-specific code, but it's not provided by this package.

Reply:
yeah, sorry, bad merge.